### ISSUE TITLE
Azure Resource Manager: While creating new connection showing a error message

### ISSUE THREAD

**Issue by vgouth** [Azure/logicapps#1251]
### Describe the Bug with repro steps 

1. Create a consumption logic app 
2. Open designer and add http request trigger 
3. Add any action from Azure Resource Manager Eg: Read a subscription 
4. Create a new connection, if there is any existing connection click on change connection and select Add new 
5. Change Authentication type to Service Principal 
6. Give connection name as mandatory field 
7. Click on create new 
8. Connection status is showing some error 

**Expected:** Connection status should show connected.

**Actual:** Connection status is showing error message.

### What type of Logic App Is this happening in? 

Consumption (Portal)

### Which operating system are you using? 

Windows

### Are you using new designer or old designer 

New Designer

### Did you refer to the TSG before filing this issue? [https://aka.ms/lauxtsg](<https://aka.ms/lauxtsg>) 

Yes

### Workflow JSON 

```json

```

### Screenshots or Videos

![Image](<https://github.com/user-attachments/assets/44991564-b24f-465e-a6ca-13d8954253a9>)

![Image](<https://github.com/user-attachments/assets/13a4af05-c72b-47aa-a635-f45c07515e4c>)

### Browser

Edge

### Additional context

**Version :** 2\.41028.1.6

**Comment by ccastrotrejo** [comment]
The API call to get the connector properties retrieve the inputs as not required

**Comment by vgouth** [comment]
Issue repro in latest canary environment
**Version:** 0.2022.1.806
![Image](https://github.com/user-attachments/assets/0962e5f8-e2f7-488a-8019-fdf2062dc870)

**Comment by vgouth** [comment]
Issue repro in latest canary test pass
**Version:** 0.2022.1.811

**Comment by vgouth** [comment]
Issue repro in latest canary test pass
**Version:** 0.2022.1.811

**Comment by vgouth** [comment]
Issue repro in latest staging test pass 

**Version:** 2.41231.2.4
![Image](https://github.com/user-attachments/assets/20b1ad7b-41e5-4fa1-b5a9-cff88429c660)

**Comment by takyyon** [comment]
@ccastrotrejo Mind looking into this please?

**Comment by ccastrotrejo** [comment]
Hi @takyyon this is something that backend engineers need to take a look at. The reason on why we are displaying the error message after creating the connection, is because its getting created without `Client ID` and `Client secret`. Therefore there is no way to get a solid connection without these. 

I checked the connector and neither of these properties come back as required. They need to enable that.

**Comment by takyyon** [comment]
@ccastrotrejo Do you mind starting a thread with backend teams for this? I'll consider this as a non-blocker for our deployment right now but let me know if you feel otherwise.

**Comment by ccastrotrejo** [comment]
@takyyon Yep I can do that, and I agree, this isn't a blocker for the deployment

**Comment by vgouth** [comment]
Issue repro in latest stage test pass
**Version:** 2.50203.1.1
![Image](https://github.com/user-attachments/assets/1213f735-15e3-49ae-bbd4-e752168e46e6)

**Comment by vgouth** [comment]
Issue repro in latest canary test pass
**Version:** 2.0.20250207.1

**Comment by vgouth** [comment]
Issue repro in latest stage test pass
**Version:** 2.50224.1.3

![Image](https://github.com/user-attachments/assets/81ca1640-6d25-47a0-a98d-0c6924cba823)

**Comment by vgouth** [comment]
Issue repro in latest staging test pass
**Version:** 2.50324.1.2
![Image](https://github.com/user-attachments/assets/fc0e507c-10a9-4cf8-9d8b-f524bc914263)

**Comment by vgouth** [comment]
Issue repro in latest staging test pass
**Version:** 2.50425.1.3

![Image](https://github.com/user-attachments/assets/53a89f5d-16a5-415b-973c-d6e731ced56c)

**Comment by vgouth** [comment]
Issue repro in latest canary test pass
**Version:** 2.0.20250512.2

![Image](https://github.com/user-attachments/assets/e4d28d2a-2b3f-4168-9b7e-e328ae07f6c3)

**Comment by vgouth** [comment]
Issue repro in latest staging test pass
Version: 2.50522.1.5

![Image](https://github.com/user-attachments/assets/41738639-6219-452f-9c3d-de118ea3ef95)

**Comment by vgouth** [comment]
Issue repro in latest staging test pass.
**Version:** 2.50616.1.1
![Image](https://github.com/user-attachments/assets/1d33912c-8262-4b9f-925f-bbeb4acde186)

**Comment by vgouth** [comment]
Issue repro in latest staging test pass
**Version:** 2.50707.1.2

<img width="968" height="476" alt="Image" src="https://github.com/user-attachments/assets/d7a8915c-436f-4c13-82b7-9647dbede155" />

**Comment by vgouth** [comment]
Issue repro in Canary Environment
Version: 2.0.20250721.1

<img width="965" height="407" alt="Image" src="https://github.com/user-attachments/assets/de00f154-f3d9-48c2-b9dd-a4d0b1f23ae2" />

**Comment by vgouth** [comment]
Issue repro in latest canary test pass
Version: 2.0.20250904.2

<img width="1051" height="516" alt="Image" src="https://github.com/user-attachments/assets/1843032b-5e85-4147-96b8-cd78265acf47" />

**Comment by vgouth** [comment]
Issue repro in latest stage test pass
Version: 2.51208.1.1


<img width="852" height="404" alt="Image" src="https://github.com/user-attachments/assets/f1b2325e-95c2-4a1b-a941-2ed5b4e54a5d" />

**Comment by vgouth** [comment]
Issue repro in latest canary test pass
**Version:** 2.0.20251217.2

<img width="888" height="444" alt="Image" src="https://github.com/user-attachments/assets/8a2a2d0d-d023-4f6b-8202-a0fcd94e77d6" />
